src/app/home/directives/home-series.component.ts

```diff
@@ -9,6 +9,9 @@ import { StoryModel } from '../../shared';
   styleUrls: ['home-series.component.css'],
   template: `
     <header *ngIf="noseries">
+      <section [publishHoverTip]="'These are your stories not associated with any Series.'" class="hover-tip">
```

Comment:
The downside is that would replace any classes defined on the parent element, rather than adding to them.  I think there are some `class.something` hostbindings in Publish somewhere already.  I ended up getting examples from [stackoverflow](http://stackoverflow.com/questions/37965647/hostbinding-and-hostlistening-what-do-they-do-and-what-are-they-for), since the docs just aren't there yet.